third-party/realdds/src/dds-trinsics.cpp

```diff
@@ -109,7 +109,27 @@ json video_intrinsics::to_json() const
         ret.principal_point.y = j[index++].get< float >();
         ret.focal_length.x = j[index++].get< float >();
         ret.focal_length.y = j[index++].get< float >();
-        ret.distortion.model = (distortion_model)j[index++].get< int >();
+        switch( j[index++].get< int >() )
```

Comment:
I think you had a function that translates rs2_distortion to realdds::distortion. Can't use it here?

Reply:
There's no knowledge of rs2_... in realdds.

Reply:
I see, it was in the proxy